src/main.rs

```diff
@@ -57,9 +57,14 @@ struct Cli {
     )]
     client_id: String,
 
+    /// Enable verbose curl output
     #[arg(short, long)]
     verbose: bool,
 
+    /// Enable curl debug output. This implicitly enables verbose output
```

Comment:
The debug output replaces the default verbose output. Although you need verbose enabled for debug, you won't see the default verbose output if you have the debug callback function setup